.github/workflows/ci-wrapper-rn.yml

```diff
@@ -97,7 +97,7 @@ jobs:
     name: Publish package release
     needs: build_test
     runs-on: macos-latest
-    if: "github.ref == 'refs/heads/master' && contains(github.event.head_commit.message, 'chore(release): publish')"
+    if: "github.ref == 'refs/heads/master' && (contains(github.event.head_commit.message, 'chore(release): publish') || contains(github.event.head_commit.message, 'chore(release): publish react-native'))"
```

Comment:
```suggestion
    if: "github.ref == 'refs/heads/master' && (contains(github.event.head_commit.message, 'chore(release): publish') || contains(github.event.head_commit.message, 'chore(release): publish react-native wrapper'))"
```

Reply:
I believe if you push a `chore(release): publish react-native wrapper` commit, this workflow may also be triggered:
https://github.com/mattrglobal/pairing_crypto/blob/ab/fix-release-pipeline/.github/workflows/ci-wrapper-wasm.yml#L111

Reply:
Ah because its contains, maybe I will use another name

Reply:
Fixed